tests/suites/test_suite_ssl.function

```diff
@@ -4912,12 +4912,17 @@ void handshake_version( int dtls, int client_min_version, int client_max_version
     handshake_test_options options;
     init_handshake_options( &options );
 
-    options.client_min_version = client_min_version;
-    options.client_max_version = client_max_version;
-    options.server_min_version = server_min_version;
-    options.server_max_version = server_max_version;
+    if ( client_min_version != TEST_SSL_MINOR_VERSION_NONE )
+        options.client_min_version = 0x0300 | client_min_version;
```

Comment:
Please change the test parameters in the *.data file to avoid the "0x0300 |". here.

Reply:
:rocket: